src/Analyzer/QueryTreePassManager.cpp

```diff
@@ -59,7 +59,7 @@ class ValidationChecker : public InDepthQueryTreeVisitor<ValidationChecker>
         if (!function->isResolved())
             throw Exception(ErrorCodes::LOGICAL_ERROR,
             "Function {} is not resolved after running {} pass",
-            function->dumpTree(), pass_name);
+            function->toAST()->dumpTree(), pass_name);
```

Comment:
For AST tree dump does not make much sense. Probably better to use query tree node `formatForErrorMessage` and prefer converted AST.